packages/static-module-record/src/static-module-record.js

```diff
@@ -60,5 +61,6 @@ export function StaticModuleRecord(source, url) {
   this.__syncModuleProgram__ = functorSource;
   this.__liveExportMap__ = liveExportMap;
   this.__fixedExportMap__ = fixedExportMap;
+  this.__usesImportMeta__ = importMetaUttered;
```

Comment:
The tentative proposed verbiage is “needs”. This is “pretend not to see” public. Which is to say, we don’t get to change it after it releases. So perhaps we should wait for a blessing on https://github.com/tc39/proposal-compartments/pull/46/ before we land this.